e2e/specs/login.e2e-spec.ts

```diff
@@ -7,6 +7,7 @@ let loginPage: LoginPage
 describe('Login', () => {
   beforeEach(() => {
     loginPage = new LoginPage();
+    browser.manage().timeouts().implicitlyWait(15000);
```

Comment:
@leszekgu this set settings globally. I would put it into `onPrepare` method in `protractor.conf.js`